cloud/aws/.env.example

```
@@ -0,0 +1,3 @@
+VAST_PEERED_VPC_ID=
+VAST_CIDR=
+VAST_AWS_REGION=
```

you are supposed to have a newline at the end of every file, but interestingly the CI didn't flag it here (probably because its an unknown `.exemple` file)
```suggestion
VAST_AWS_REGION=

```

---

Notably not an empty line, but rather a newline at the end. We don't have a CI job that enforces the .editorconfig file in the repo.

---

Done.